test/e2e/scenarios/bare-metal-testdriver.go

```diff
@@ -83,8 +83,18 @@ func (d *baremetalDriver) GetDriverInfo() *testsuites.DriverInfo {
 
 // SkipUnsupportedTest is implementation of TestDriver interface method
 func (d *baremetalDriver) SkipUnsupportedTest(pattern testpatterns.TestPattern) {
-	if pattern.VolType == testpatterns.InlineVolume || pattern.VolType == testpatterns.PreprovisionedPV {
-		e2eskipper.Skipf("Baremetal Driver does not support InlineVolume and PreprovisionedPV -- skipping")
+	if !d.needAllTests {
+		if pattern.VolMode == corev1.PersistentVolumeBlock {
```

Comment:
Please add comments here why we skip this tests.

Reply:
comments added